src/arrayMethodSort.js

```diff
@@ -5,7 +5,39 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    let callback = compareFunction;
+    let countArrayItterations;
+
+    if (arguments.length === 0) {
+      callback = (prev, curr) => {
+        if (String(prev) > String(curr)) {
```

Comment:
You can just return `String(prev) > String(curr)` from your `compareAsStrings` function